src/pykeen/experiments/nodepiece/galkin2022_nodepiece_codexl.yaml

```diff
@@ -0,0 +1,43 @@
+metadata:
+  title: Learn CoDEx-Large Dataset with NodePiece + RotatE as described by Galkin et al., 2022
+pipeline:
+  dataset: CoDExLarge
+  evaluator_kwargs:
+    filtered: true
+  loss: BCELoss
```

Comment:
Did you mean `BCEWithLogitsLoss`?

Reply:
Ah right, my bad :/

Reply:
fixed in https://github.com/pykeen/pykeen/pull/771/commits/474e64b064615e48e391271e303fc37a220e07ae